docs/release-notes/release-notes-0.1.0.md

```diff
@@ -0,0 +1,31 @@
+# About this release
+This release is intended to be compatible with the [aeternity epoch 0.16.0](https://github.com/aeternity/epoch/releases/tag/v0.16.0) release. It includes all major features in the state of the development at that point with a few differences - state channel P2P logic isn't implemented and smart contracts don't support operations native to the Sophia language (Solidity smart contracts are still supported).
```

Comment:
we don't need to add "state channel P2P logic isn't implemented", this is not needed to be compatible